infra/hooks/postdeploy.ps1

```diff
@@ -0,0 +1,13 @@
+# PowerShell syntax
+echo "Configuring EventGrid subscription for API Center"
+
+# Sleep for 2 minutes to allow the function app to be deployed
+Start-Sleep -s 120
+
+$output = azd env get-values  
+$resourceGroupName = ($output -split "`n" | Where-Object { $_ -match 'RESOURCE_GROUP_NAME' }).Split("=")[1]
+$azureApiCenterId = ($output -split "`n" | Where-Object { $_ -match 'AZURE_API_CENTER_ID' }).Split("=")[1]  
+$azureFunctionName = ($output -split "`n" | Where-Object { $_ -match 'AZURE_FUNCTION_NAME' }).Split("=")[1] 
+$azureFunctionId = az functionapp function show --name $azureFunctionName --function-name apicenter-analyzer --resource-group $resourceGroupName --query "id" --output tsv
+
+az eventgrid event-subscription create --name MyEventSubscription --source-resource-id "$azureApiCenterId" --endpoint "$azureFunctionId" --endpoint-type azurefunction --included-event-types Microsoft.ApiCenter.ApiDefinitionAdded Microsoft.ApiCenter.ApiDefinitionUpdated
```

Comment:
Instead of giving it a name of `MyEventSubscription`, I'd more towards having a meaningful name for the event grid subscription.

Because this sub is only looking for both `Microsoft.ApiCenter.ApiDefinitionAdded` and `Microsoft.ApiCenter.ApiDefinitionUpdated`, can we give it a name like `on-api-definition-added-or-updated`?